apiserver/facades/client/modelconfig/modelconfig.go

```diff
@@ -197,3 +197,19 @@ func (c *ModelConfigAPI) SLALevel() (params.StringResult, error) {
 	result.Result = level
 	return result, nil
 }
+
+// Sequences returns the model's sequence names and next values.
+func (c *ModelConfigAPI) Sequences() (params.ModelSequencesResult, error) {
```

Comment:
To be safe, let's rev the facade version? I have a feeling the Python client library has tests which validate that their api caller layer methods match those exposed by Juju for a given facade version.

Reply:
Facade now v2